supervisor/shared/usb/host_keyboard.c

```diff
@@ -78,6 +78,26 @@ struct keycode_mapper {
 #define CURSOR_INS "\e[2~"
 #define CURSOR_DEL "\e[3~"
 
+// https://learn.microsoft.com/ru-ru/windows/console/console-virtual-terminal-sequences
+// https://aperiodic.net/phil/archives/Geekery/term-function-keys/
+#define F1      "\eOP"
+#define F2      "\eOQ"
+#define F3      "\eOR"
+#define F4      "\eOS"
+#define F5      "\e[15~"
```

Comment:
Oddly, my documentation ("man 7 urxvt") has "\e[11~" through "\e14~" for F1..F4.  "\eOP" .. "\eOS" are for KP_F1..KP_F4, which do not exist on traditional PC keyboards.  **however** both xterm and xfce4-terminal are behaving like you've implemented here, so I'm sure it's fine.

Reply:
done. I've change to "\e[11~" through "\e14~"